app/src/main/java/com/example/genericapplication/MainActivity2.kt

```diff
@@ -24,7 +25,11 @@ import kotlinx.android.synthetic.main.activity_main2.*
 class MainActivity2 : AppCompatActivity() {
     private var player: ExoPlayer? = null
     private var adsLoader: ImaAdsLoader? = null
+    private var videoUrl: String? = null
     private var mediaUrl: String? = null
+    private var videoIdentifier: String? = null
```

Comment:
@DiegoA-coder I consider that unnecessary the definition of these variables here because the are only used on the `buildMediaUrl` function

Reply:
I use it in getMediaUrl I don't know if it is the best implementation

Reply:
@DiegoA-coder perfect! thank you for the explanation